search.py

```diff
@@ -94,6 +98,8 @@ def get_title(path):
     """
     tree, _ = prepare_tree(path)
     title_stmt = get_elements(tree, 'titleStmt')
+    if not title_stmt:
+        return "Title Not Found"
```

Comment:
A better design is to return None, then display "Title Not Found" on the front end.